<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalDetails</class>
 <widget class="QWidget" name="SignalDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignalDetails</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Signal</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="aliasLabel">
          <property name="text">
           <string>alias</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="aliasEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="commentLabel">
          <property name="text">
           <string>comment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="commentEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="startLabel">
          <property name="text">
           <string>start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="startEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="lengthLabel">
          <property name="text">
           <string>length</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lengthEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="typeLabel">
          <property name="text">
           <string>type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="typeEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="unitLabel">
          <property name="text">
           <string>unit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="unitEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="minValueLabel">
          <property name="text">
           <string>min_value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="minValueEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="maxValueLabel">
          <property name="text">
           <string>max_value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maxValueEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="muxLabel">
          <property name="text">
           <string>mux</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="muxEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="muxCountLabel">
          <property name="text">
           <string>mux_count</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="muxCountEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="byteOrderLabel">
          <property name="text">
           <string>byte_order</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="byteOrderEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="scaleLabel">
          <property name="text">
           <string>scale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="scaleEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="offsetLabel">
          <property name="text">
           <string>offset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="offsetEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="saveSignal">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="signalDeleteButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
